platform/jobs/edxPlatformAccessibilityPr.groovy

```diff
@@ -107,11 +113,14 @@ secretMap.each { jobConfigs ->
                 orgWhitelist(ghprbMap['orgWhiteList'])
                 extensions {
                     commitStatus {
-                        context('jenkins/a11y')
+                        context(jobConfig['context'])
                     }
                 }
             }
         }
+
+        configure GHPRB_WHITELIST_BRANCH(jobConfig['whitelistBranchRegex'])
```

Comment:
and for my own clarification, is it not available in this *version* of the plugin, or is it not-available at all?